sdk/schemaregistry/Azure.Data.SchemaRegistry/tests/Properties/Resources.resx

```diff
@@ -0,0 +1,101 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
```

Comment:
What is the deal with this file?

Reply:
I don't know, I think it got generated by something. I'll delete it